### DISCUSSION TITLE
[TAB] Change tabs position depends on device

### DISCUSSION THREAD

**Discussion by yangetze** [Megabit/Blazorise#4456] [Ideas]
I was thinking that it would be interesting that depending on the device, it changes the position of the tabs i.e. I'm using a TabPosition="TabPosition.Start", and I want to change the position of the tab if it's mobile, to TabPosition.Top.

Maybe something like, TabPosition="TabPosition.Start.OnDesktop.Top.OnMobile".

**Comment by stsrki** [comment]
It's not a bad idea on first look. But it would not work, unfortunately. Every tab position has a different HTML structure.

Other fluent utilities use special CSS class names that do all the heavy lifting with media queries. But the same HTML structure is still the same.
